bcgov/onroutebc
# ORV2-1269, ORV2-1268: updated constraints name to follow moti standard and updat…

…ed receipt not null columns



# Description

This PR is to update v_7_ddl.sql constraints name to follow MOTI data standard, as well as update ORBC_RECEIPT not null columns.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Test steps:
1. Run v_7_ddl.sql to refresh the database.
2. Verify the constraint names on ORBC_TRANSACTION, ORBC_PERMIT_TRASACTION, ORBC_VT_PAYMENT_MOTHOD and ORBC_RECEIPT. (MOTI data standard: https://projects.sp.th.gov.bc.ca/_layouts/15/WopiFrame.aspx?sourcedoc=/Standards%20and%20Guidelines/Data%20Architecture%20Standards%20V2.00.docx&action=default
)
4. Verify not null columns on ORBC_RECEIPT, expect RECEIPT_ID, RECEIPT_NUMBER, TRANSACTION_ID, and RECEIPT_DOCUMENT_ID are not null.




## Checklist






## Further comments




---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend - Vehicles](https://onroutebc-571-backend-vehicles.apps.silver.devops.gov.bc.ca/) available
[Backend - DOPS](https://onroutebc-571-backend-dops.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://onroutebc-571-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge-main.yml)
